contracts/lines/AxelarLine.sol

```diff
@@ -28,6 +28,10 @@ contract AxelarLine is BaseMessageLine, AxelarExecutable, Ownable {
         GAS_SERVICE = IAxelarGasService(_gasReceiver);
     }
 
+    function getLineInfo() external pure returns (string memory) {
```

Comment:
What is the purpose of lineinfo?

Reply:
For dapp to know what this line is, and do checking on the info.

Reply:
Is it enough to only provide the line address? Because the dapp selects the line according to the provider, and the dapp knows who the provider of this line is

Reply:
@hujw77 Do you have any suggestions? I think the dapp will use like this
```sol
address axelarLine = "0xabcd"
```
And then check if the line is axelar
```
if fromLineAddress == axelarLine
```
Will they use the lineInfo like "Axelar"?

Reply:
I think it would be more appropriate to change the function name to "lineProvider".

Reply:
Should lineProvider be stored in metadata?

Reply:
Yes, it's better.

Reply:
So we should remove lineInfo here and handle it in metadata later. https://github.com/darwinia-network/darwinia-msgport/issues/33

Reply:
but for dapp view, the concept of `line` equals to the messaging provider. That's why I did't use the term `provider` here(I used `provider` in the `dock` context).

Reply:
line metadata?